### ISSUE TITLE
Aurora palace’s interior doesn’t generate properly

### ISSUE THREAD

**Issue by generic-user-acc** [TeamTwilight/twilightforest#636]
Forge version: 1.12.2 - 14.23.4.2765
Twilight Forest version: 3.8.689
Link to crash log: none

Steps to reproduce:
1. Go inside aurora palace

What I expected to happen: The rooms generate properly and the ways to get from one room to another aren’t blocked

What happened instead: Rooms generate improperly. Sometimes stairs go into blocked ceilings, doorways are blocked by other features, and overall many places have blocks in places where they shouldn’t be, preventing the player from travelling through and making exploring the palace difficult.

Can’t get screenshots right now but let me know if you require them.

**Comment by Andromander** [comment]
Duplicate of #71 and will be addressed in #496

**Comment by Killer-Demon** [comment]
^
